ROADMAP.md

```diff
@@ -0,0 +1,128 @@
+# Roadmap
+
+Last updated: 2024-03-12
+
+## Background
+
+This document sets out the plan for the evolution of the Data Safe Haven codebase.
+It covers both short term-goals, which are organised in sprint milestones, and long-term goals, which are expected without a particular time frame.
+It also collects desirable features which are unplanned or which we have decided not to pursue yet.
+
+The [short-term plans](#short-term) aim to give a clear indication of current work and when it may be finished.
+We hope that a focus on defining releases will encourage development to balance new features with other improvements and address the needs of users.
+
+[Long-term goals](#long-term) give reassurance that we have committed to certain changes when we are unable to give an estimation of when they will be ready.
+
+Finally, [desired features](#desired-features) is a way for us to express developments we would like to pursue but have decided not to do yet.
+This is a good indication of where we would like the project to go and areas we would be interested in collaborating.
+
+## Governance
+
+This document is written by the code's [development lead](https://github.com/JimMadge) with input from the development team.
+
+When deciding on what features to include in the roadmap, and where to include them we consider, in no particular order,
+
+- The maintainability of the code (looking after the developers)
+- Supporting users at The Alan Turing Institute (a significant user who we have a close relationship with)
+- Supporting all users we are in contact with
+- Integrating with, and building compatibility for, common infrastructure in the UK TRE landscape
+
+These factors are balanced when making decisions.
+There is no formal process for assigning scores for the above factors, or the weight each should be given.
+In general, we currently consider the needs of our colleagues at The Alan Turing institute as our single most important user.
```

Comment:
Yes, I thought it was important to be honest.

The process is _we_ decide what to do so I thought it was good to explain what we care about.
Also, it could change later.